{"74554f66f8b6eac9934f0a1196eea50a":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"ae59c3c051385c31","reportUrl":"../1/index.html#testresult/ae59c3c051385c31","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532690,"stop":1706852532690,"duration":0}}]},"a65093e250e3fda1fbd2d8ea50d0be36":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"22e727e891beb581","reportUrl":"../1/index.html#testresult/22e727e891beb581","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852533233,"stop":1706852533233,"duration":0}}]},"21b284cc27c0ccaa5f5d5cdf061ebad7":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"27c1fd042dbd1a44","reportUrl":"../1/index.html#testresult/27c1fd042dbd1a44","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532410,"stop":1706852532410,"duration":0}}]},"c6e1fe33e5c7b5412729919b47bb73db":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"2ca24d992051a","reportUrl":"../1/index.html#testresult/2ca24d992051a","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532227,"stop":1706852532227,"duration":0}}]},"f13f13d43e77423394b7d8732dca9584":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"e1667dc8e84fa74","reportUrl":"../1/index.html#testresult/e1667dc8e84fa74","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532897,"stop":1706852532897,"duration":0}}]},"c1fa394b1acadb4f5087a77b057e8086":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"c7fde30a4edafd7","reportUrl":"../1/index.html#testresult/c7fde30a4edafd7","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852533122,"stop":1706852533122,"duration":0}}]},"1918a52aa2987e57a021576c386bac3a":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"57f4a4f7c8cd776e","reportUrl":"../1/index.html#testresult/57f4a4f7c8cd776e","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852533345,"stop":1706852533345,"duration":0}}]},"02dbe5a571b0528f1bb26ff8441dccf0":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"eab13da1262b2a6e","reportUrl":"../1/index.html#testresult/eab13da1262b2a6e","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532597,"stop":1706852532597,"duration":0}}]},"6475c0415291b1b14579bea29e2973d3":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"f45aad97b731c61a","reportUrl":"../1/index.html#testresult/f45aad97b731c61a","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532505,"stop":1706852532505,"duration":0}}]},"7d1d78131780ca25a1ed9624f432af5f":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"391b88578e334c68","reportUrl":"../1/index.html#testresult/391b88578e334c68","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852515696,"stop":1706852515696,"duration":0}}]},"9d0f73ef394b467c436214e041bf9a41":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"40a214844fb63468","reportUrl":"../1/index.html#testresult/40a214844fb63468","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532786,"stop":1706852532786,"duration":0}}]},"0b9ac6741417e5c6d6502999f717d8c2":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"74fa574b4883d894","reportUrl":"../1/index.html#testresult/74fa574b4883d894","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532311,"stop":1706852532311,"duration":0}}]},"a9234975d44c068c7f69d9fd5ed48f0a":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"94201fd6e08c3f89","reportUrl":"../1/index.html#testresult/94201fd6e08c3f89","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852532145,"stop":1706852532145,"duration":0}}]},"451b6dc509e1a540cc87d6a6b45914c5":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"3319f1705f0b16f0","reportUrl":"../1/index.html#testresult/3319f1705f0b16f0","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002001383E960>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","time":{"start":1706852533011,"stop":1706852533011,"duration":0}}]}}